pyproject.toml

```diff
@@ -38,37 +30,46 @@ cu12 = [
     "nvidia-cuda-nvrtc-cu12",
     "nvidia-nvjitlink-cu12"
 ]
+cu13 = [
+    "cuda-bindings==13.0.*",
+    "cuda-core==0.3.*",
+    "cuda-python==13.0.*",  # supports all CTK 12.x
+    "nvidia-cuda-nvcc-cu13",  # for libNVVM
+    "nvidia-cuda-runtime-cu13",
+    "nvidia-cuda-nvrtc-cu13",
+    "nvidia-nvjitlink-cu13"
```

Comment:
The scheme has changed for CUDA 13 where this is now `nvidia-cuda-nvcc=13.*` for example